package.json

```diff
@@ -11,26 +11,27 @@
     "url": "git@github.com:marklogic-community/grove-react-ui.git"
   },
   "dependencies": {
-    "bootstrap": "^3.4.1",
-    "bootstrap-material-design": "^0.5.10",
+    "bootstrap": "^4.5.0",
+    "bootstrap-material-design": "^4.1.3",
     "codemirror": "^5.54.0",
     "isomorphic-fetch": "^2.2.1",
-    "lodash": "^4.17.11",
-    "query-string": "^6.12.1",
-    "react": "^16.8.6",
-    "react-bootstrap": "^0.31.2",
-    "react-codemirror2": "^5.1.0",
-    "react-dom": "^16.8.6",
-    "react-json-tree": "^0.11.0",
-    "react-redux": "^5.1.1",
-    "react-router-bootstrap": "^0.24.4",
-    "react-router-dom": "^4.2.2",
+    "lodash": "^4.17.15",
+    "query-string": "^6.13.0",
+    "react": "^16.13.1",
+    "react-bootstrap": "^1.0.1",
+    "react-codemirror2": "^7.2.1",
+    "react-dom": "^16.13.1",
+    "react-json-tree": "^0.11.2",
+    "react-redux": "^7.2.0",
+    "react-router-bootstrap": "^0.25.0",
```

Comment:
We believe this is no longer used with your refactor.  Can you also remove it from dependencies?  Dave Grant caught this during a team review of this PR today.